### PR TITLE
Release 20210512

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] All new files are linked in indices
- [x] Front matter is present and correct on Markdown (not HTML) pages
- [x] Bullets and numbered lists are correct
- [x] Correct Markdown and formatting
- [x] All content reviewed

-----
[View rendered careers/content-editor.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210512/careers/content-editor.md)
[View rendered who-we-are/team/ben-cerveny.md](https://github.com/publiccodenet/publiccode.net/blob/release-20210512/who-we-are/team/ben-cerveny.md)

### CODE REVIEW

**Comment by Ainali** [comment]
Is the new position linked to from anywhere?

File: careers/content-editor.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+type: Position
+excerpt:  Tell stories about public code's impact that everyone can understand
```
Comment (Ainali) [inline careers/content-editor.md:3]:
```suggestion
excerpt: Tell stories about public code's impact that everyone can understand
```

**Comment by ElenaFdR** [comment]
@ainali The [careers index page](https://github.com/publiccodenet/publiccode.net/blob/develop/careers/index.md) pulls open positions automatically. It's not linked to from anywhere else (yet!)

**Review by Ainali — APPROVED** [review]
Alright, this looks really good!

**Pull request merged**